src/plugins/discover/public/application/main/components/layout/discover_histogram_layout.tsx

```diff
@@ -61,15 +64,22 @@ export const DiscoverHistogramLayout = ({
       }
       css={histogramLayoutCss}
     >
-      <DiscoverMainContent
-        {...mainContentProps}
-        stateContainer={stateContainer}
-        dataView={dataView}
-        isPlainRecord={isPlainRecord}
-        // The documents grid doesn't rerender when the chart visibility changes
-        // which causes it to render blank space, so we need to force a rerender
-        key={`docKey${hideChart}`}
-      />
+      <>
+        <SelectedVSAvailableCallout
```

Comment:
Can we move this to `discover_main_content.tsx` below `ErrorCallout` instead?

Reply:
Done https://github.com/elastic/kibana/pull/165557/commits/15b234f25bdbcbb03292cf9331ff0468e0aa4351